containers/src/Kokkos_DualView.hpp

```diff
@@ -91,6 +91,25 @@ namespace Kokkos {
  *     behavior.  Please see the documentation of Kokkos::View for
  *     examples.  The default suffices for most users.
  */
+
+namespace Impl {
+
+#ifdef KOKKOS_ENABLE_CUDA
+
+inline const Kokkos::Cuda& get_cuda_space(const Kokkos::Cuda& in) { return in; }
+
+inline Kokkos::Cuda get_cuda_space() {
+  return Kokkos::Cuda(Kokkos::Impl::cuda_get_deep_copy_stream());
+}
+
+template <typename Arg>
+inline Kokkos::Cuda get_cuda_space(Arg&) {
+  return Kokkos::Cuda(Kokkos::Impl::cuda_get_deep_copy_stream());
```

Comment:
... and here?

Reply:
See https://github.com/kokkos/kokkos/pull/3822#discussion_r590649342